mlrun/feature_store/steps.py

```diff
@@ -294,7 +399,10 @@ def do(self, event):
                 timestamp = event[self.timestamp_col]
             except Exception:
                 raise ValueError(f"{self.timestamp_col} does not exist in the event")
```

Comment:
I know its not from your PR, but its not healthy to catch all possible exceptions and convert them to one. Try to give it a look and see if you get what they mean. Also, the `ValueError` should be `mlrun.errors.Invalid...`

Reply:
The most reasonable error here is KeyError so I changed it.